config/management/src/secure_backend.rs

```diff
@@ -147,6 +149,7 @@ pair: "k0=v0;k1=v1;...".  The current supported formats are:
         an optional namespace: "namespace=NAMESPACE"
         an optional server certificate: "ca_certificate=PATH_TO_CERT"
     GitHub: "backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_TOKEN"
+        an optional branch: "branch=BRANCH", defaults to master
```

Comment:
So the caller has to explicitly specify the branch? What about defaulting to `genesis` when doing `genesis` or `create-waypoint` operations?

Reply:
I thought about this and would prefer we introduce this nuance to the tool as testing it from the tooling side is a bit more cumbersome, would require additional testing, and just be a pain in the butt... whereas fixing it in your script is a lot "easier" ... though happy to discuss more if that's not the case.